src/messages/formatters.ts

```diff
@@ -222,3 +222,23 @@ export const TimestampStyles = {
  * The possible values, see {@link TimestampStyles} for more information.
  */
 export type TimestampStylesString = typeof TimestampStyles[keyof typeof TimestampStyles];
+
+/**
+ * An enum with all the available faces from Discord's native slash commands
+ */
+export enum Faces {
+	/**
+	 * ¯\\_(ツ)_/¯
+	 */
+	Shrug = '¯\\_(ツ)_/¯',
```

Comment:
We need 3 slash because 1 is to render on discord, an other should escape the other on discord and the third should escape them in the js string.
```suggestion
	Shrug = '¯\\\_(ツ)\\_/¯',
```
Should be changed in tests as well.

Reply:
Nope we don’t. If you add the first one, prettier will ask you to remove it and adding the second two makes it so that it doesn’t match with the content from Discord’s shrug

Reply:
Ooh you're right, sorry!